.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Please describe the problem this feature would solve.**
+Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Has the feature been requested before?**
+If so, please provide a link to the issue.
+
+**Additional context**
+<!-- Add any other context or screenshots about the feature request here. -->
```

Comment:
@sharma-shruti , I need you to think about what could be the thought of the end user like @Ankita-1211 when she's creating the issue.

Please think of more user-and-usage centric points and ask for a review from @Ankita-1211  as well.